bats/helper/common.bash

```diff
@@ -5,10 +5,33 @@ if [ -z "$BATS_TMPDIR" ]; then
 	mkdir $BATS_TMPDIR
 fi
 
-nativevar DOLT_ROOT_PATH $BATS_TMPDIR/config-$$ /p
-dolt config --global --add user.name "Bats Tests"
-dolt config --global --add user.email "bats@email.fake"
-dolt config --global --add metrics.disabled true
-
 nativebatsdir() { echo `nativepath $BATS_TEST_DIRNAME/$1`; }
 batshelper() { echo `nativebatsdir helper/$1`; }
+
+set_dolt_user() {
+	dolt config --global --add user.name "$1" > /dev/null 2>&1
+	dolt config --global --add user.email "$2" > /dev/null 2>&1
+}
+
+setup_common() {
+	export PATH=$PATH:~/go/bin
+	export NOMS_VERSION_NEXT=1
+	cd $BATS_TMPDIR
+	# Append the directory name with the pid of the calling process so
+	# multiple tests can be run in parallel on the same machine
+	mkdir "dolt-repo-$$"
+	cd "dolt-repo-$$"
+
+	SKIP_INIT=$1
+	if [ "$SKIP_INIT" != true ] ; then
```

Comment:
Please do let me know if you know a better way to write this. I tried a few, actually...

Reply:
What if you made two functions? One called setup_common() and one called setup_no_dolt() and make the setup_no_dolt() call setup_common(). Passing the variable is a little janky. The above seems to be more readable.

Reply:
I like that, thanks.